pagopa/io-app
# feat(IT Wallet): [SIW-1090] Add NFC enable instructions screen

> [!WARNING]
> This PR depends on https://github.com/pagopa/io-app/pull/5745

## Short description
This PR adds the screen to display the instructions to enable the NFC for Android users. 

## List of changes proposed in this pull request
- Added `ItwIdentificationNfcInstructionsScreen` component
- Added required locale keys

## How to test
In order to test this PR on dev environment, make sure to add this flag to your .env:
```
# Enable CIE login flow with emulator and dev server
CIE_LOGIN_WITH_DEV_SERVER_ENABLED=YES
```
#### On a simulator
Go to **Profile > IT Wallet > Issuing**, clicking on CIE + PIN you should be able to see the NFC instructions screen.
#### On a physical Android device
Disable the NFC, then go to **Profile > IT Wallet > Issuing**, clicking on CIE + PIN you should be able to see the NFC instructions screen.

## Preview
<img src="https://github.com/pagopa/io-app/assets/6160324/eb608463-8eb5-4bd1-93db-6e77302e80a5" width="250" />
